hubble/resources.py

```diff
@@ -1,9 +1,22 @@
 import falcon
 import json
+import boto3
+import time
+import uuid
 
+from hubble.models import Event
 from hubble import models
 
 
+s3_region_name = 'ca-central-1'
```

Comment:
Do we want to make this configurable?

Reply:
Eventually, I think we'll want to let people setup their own S3 instances since I'm not likely to leave this one running forever, that said, setting up S3 is not trivial for lots of people so a default isn't a bad idea.